helix-term/src/main.rs

```diff
@@ -114,6 +114,18 @@ FLAGS:
         return Ok(0);
     }
 
+    #[cfg(feature = "jujutsu")]
+    if let Some(diff_base) = args.jujutsu_diff_base {
+        match std::fs::copy(diff_base.base, &diff_base.target) {
+            Ok(_) => (),
+            Err(err) if err.kind() == std::io::ErrorKind::NotFound => {
+                std::fs::copy(diff_base.current, diff_base.target)?;
+            }
+            Err(err) => return Err(err.into()),
+        }
+        return Ok(0);
+    }
+
```

Comment:
this could be done in a wrapper script I don't think that belongs in core

Reply:
You mean in a personal script for it ? it would work yes, I made my tests simply using `cp` but it means relying on another third-party binary.

I can change it to use `cp` on Unix and `copy.exe` on windows if wanted

Reply:
yeah I mant as in a personal script not upstream in this repo. `cp`/`copy.exe` is available everywhere I don't think its an issue relying on that.

Reply:
That'll simply the diff, will do!

Reply:
Done